ci/editorconfig.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-set -eu
+set -euo pipefail
```

Comment:
The reason this was not in there was that `-o pipefail` is not supported in POSIX shell, only in `bash` (and `zsh` I guess). On the testing docker image there is no `bash`. I think it's good if you add it there, so we can use `pipefail` in all the scripts.

Reply:
> .. (`sed s#^ *//#--#g`), I think it's good to add it too.

I slightly improved that by allowing an arbitrary number of whitespace before and after closing comment blocks. Also as this is the end of a comment block, I decided to remove that completely

`'s#^\s*\*\/\s*##g'`

Reply:
Apparently you already merged a PR for bash https://github.com/citusdata/the-process/pull/36 . The only missing step is to actually build and push the images to docker hub. I will update the image and change the interpreter to bash to be able to use the `pipefail` option

Reply:
Alright, well if you can push the images. feel free to update the ci scripts to all use `#!/bin/bash` and `-o pipefail`